docs/nidigital/class.rst

```diff
@@ -188,7 +188,7 @@ burst_pattern
 
     .. py:currentmodule:: nidigital.Session
 
-    .. py:method:: burst_pattern(site_list, start_label, select_digital_function, wait_until_done, timeout)
+    .. py:method:: burst_pattern(site_list, start_label, select_digital_function=True, wait_until_done=True, timeout=10.0)
```

Comment:
timeout is the wrong type... have you addressed those converters or is there still a pending task about it?